src/couch_epi.erl

```diff
@@ -21,12 +21,13 @@
     by_key/1, by_key/2, by_source/1, by_source/2,
     keys/1, subscribers/1]).
 
+-export([is_configured/3]).
+-export([is_defined_by_app/4]).
+
 %% apply
 -export([apply/5]).
 -export([any/5, all/5]).
 
--export([is_configured/3]).
```

Comment:
> @kxepal: Accident export?

No. I just moved the export up. So it is in `%% queries and introspection` section as it should be.